### PR TITLE
Upgrade samples to player-v8

### PR DESCRIPTION
**Pull request by stonko1994**
## Description
This PR updates all samples to the current stable Player v8 version.

_The big number of changes result from unifying some code formats._

New Samples:
* react
* vue

### CODE REVIEW

File: adaptation/bitrateGraphs.js [hunk 1]
```diff
@@ -43,24 +43,24 @@ BitrateGraphs.prototype.init = function() {
   google.charts.load('current', { 'packages': ['corechart'] });
   google.charts.setOnLoadCallback(this.drawLineChart.bind(_this));
 
-  this.player.addEventHandler('onDownloadFinished', function(data) {
-    if (data.downloadType === 'media' && data.mimeType.indexOf('video') >= 0 && data.size > 1000) {
+  this.player.on('downloadfinished', function(data) {
```
Comment (protyposis) [inline adaptation/bitrateGraphs.js]: Use of strings is discouraged. You should use the enum instead: `bitmovin.player.PlayerEvent.DownloadFinished`

File: webpack-demo/src/index.js [hunk 2]
```diff
@@ -1,31 +1,23 @@
...
 
-var player = bitmovinplayer('player');
+var player = new Player(document.getElementById('player'), conf);
 
-player.setup(conf).then(function (value) {
-  playerui.UIManager.Factory.buildDefaultUI(player);
-  console.log('Successfully created Bitmovin Player instance'); // Success!
-}, function(reason) {
-  console.log('Error while creating Bitmovin Player instance'); // Error!
+UIFactory.buildDefaultUI(player, {});
```
Comment (protyposis) [inline webpack-demo/src/index.js]: The config is optional and can be left away.

File: adaptation/bitrateGraphs.js (hunk 1 above)
Comment (stonko1994) [reply adaptation/bitrateGraphs.js]: I thought that too but than i found the sample `events/eventConstants` which was the only one where the Enum was used.

But if it is the preferred way i will change it everywhere (and delete the `eventConstants` sample?)?

Comment (dweinber) [reply adaptation/bitrateGraphs.js]: That makes sense. I think the `eventConstants` was created when we introduced the event constants in the player 😉

**Comment by stonko1994** [comment]
@dweinber @protyposis do we need a branch where we have 'support-v7' samples?

File: adaptation/preferredStartupQuality.js [hunk 3]
```diff
@@ -68,16 +68,18 @@ var PreferredStartupQuality = function (player, preferredStartupQualityBps, star
...
+
+    player.load(conf.source).then(function() {
+        console.log('Successfully loaded source');
+    }, function () {
+        console.log('Error while loading source');
     });
 
     this.remove = function () {
-        player.removeEventHandler('onReady', onReady);
+        player.off('ready', onReady);
```
Comment (protyposis) [inline adaptation/preferredStartupQuality.js]: Use the enum here as well :)

**Comment by protyposis** [comment]
The branch can always be created at a later time if needed.

**Pull request merged**